docs/sphinx/source/index.rst

```diff
@@ -221,6 +221,20 @@ The most frequently used functions are:
      Outputs: DataFrame of production loss and availability metrics
      '''
 
+Documentation
+-------------
+
+Some RdTools function parameters can take one of several types.  For example,
+the ``albedo`` parameter of :py:class:`TrendAnalysis` can be a static value like
+``0.2`` or a time-varying ``pandas.Series``.  To indicate that a parameter can
+take one of several types, we document them using the type alises listed below:
+
+.. glossary ::
+
+   ``numeric``
+      scalar, ``numpy.array``, ``pandas.Series``. Typically int or float dtype.
```

Comment:
1) I copied this from pvlib's definition as a starting place.  Not sure we want `numpy.array` here.
2) It's a little silly to have a `glossary` with only one entry (for now, anyway), but this is the most convenient way I know of to get a linkable entity like `:term:` to use in the alias target.